samples/TempLCD/GroveLCDLib.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2016, Intel Corporation.
+// JavaScript library for the Grove LCD
+
+export function GroveLCD() {
```

Comment:
@grgustaf, well, I don't agree with that, the I2C.js demo is exactly what that's for, this patch is show the demo of multi-js, the bmp280 is a indeed good example because we don't have drivers for it, so it gives application an example how to extend the it.  Now if this I2C talks with another LCD like the ones we ordered that does more fancy things, that would be cool.